pkg/cli/admin/upgrade/recommend/recommend.go

```diff
@@ -343,9 +343,12 @@ func (o *options) Run(ctx context.Context) error {
 					}
 					unaccepted := issues.Difference(accept)
 					if unaccepted.Len() > 0 {
+						if !o.quiet {
+							fmt.Println(o.Out)
```

Comment:
This looks wrong, this prints whatever the string representation of `o.Out` is (output stream)

Reply:
Thanks; this is what was causing [the `&{0xc0001520c0}`][1].  Fixed to use `Fprintln` in 4e9346615 -> f3de5e0a6.

[1]: https://github.com/openshift/oc/pull/2045#issuecomment-2986382436